rke/k8s_rke_system_images.go

```diff
@@ -6957,7 +7045,7 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			AciGbpServerContainer:     "noiro/gbp-server:5.1.1.0.1ae238a",
 			AciOpflexServerContainer:  "noiro/opflex-server:5.1.1.0.1ae238a",
 			PodInfraContainer:         "rancher/mirrored-pause:3.6",
-			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.1.0-rancher1",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.2.0-rancher1",
```

Comment:
For the line 7013 above we wanted to change comment from `// Enabled in v2.6.5` to `// Enabled out of band post v2.6.4`. Same is applicable to other versions introduced in https://github.com/rancher/kontainer-driver-metadata/pull/884.